src/os/bluestore/BlueStore.h

```diff
@@ -407,11 +414,15 @@ class BlueStore : public ObjectStore,
     void _rm_buffer(BufferCacheShard* cache, Buffer *b) {
       _rm_buffer(cache, buffer_map.find(b->offset));
     }
-    std::map<uint32_t, std::unique_ptr<Buffer>>::iterator
+    std::map<uint64_t, std::unique_ptr<Buffer>>::iterator
     _rm_buffer(BufferCacheShard* cache,
-		    std::map<uint32_t, std::unique_ptr<Buffer>>::iterator p) {
+		    std::map<uint64_t, std::unique_ptr<Buffer>>::iterator p) {
       ceph_assert(p != buffer_map.end());
+      std::stringstream ss;
+      ss << __func__ << std::hex << " 0x" << p->first << "~"
+                            << p->second->length << std::dec << std::endl;
       cache->_audit("_rm_buffer start");
+      cache->_audit(ss.str().c_str());
```

Comment:
Doing audit 2 times in a row. 
Having custom name for audit reason is nice, but then wrap it with #ifdef DEBUG_CACHE.